<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="frequencyBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Indentor Frequency</string>
    </property>
    <widget class="QLCDNumber" name="frequencyDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="frequencySlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="decrButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="incrButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="amplitudeBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Amplitude</string>
    </property>
    <widget class="QDoubleSpinBox" name="amplitudeSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Logging</string>
    </property>
    <widget class="QPushButton" name="saveLogButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Log File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearDataButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Mode</string>
    </property>
    <widget class="QComboBox" name="controlModeBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Current</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Position</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Open Loop</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Recieved Data</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="timestampLabel">
        <property name="text">
         <string>Timestamp</string>
        </property>
        <property name="buddy">
         <cstring>timestampLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="timestampLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="positionLabel">
        <property name="text">
         <string>Position</string>
        </property>
        <property name="buddy">
         <cstring>positionLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="positionLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="currentLabel">
        <property name="text">
         <string>Current</string>
        </property>
        <property name="buddy">
         <cstring>currentLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="currentLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="accelerationLabel">
        <property name="text">
         <string>Acceleration</string>
        </property>
        <property name="buddy">
         <cstring>accelerationLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="accelerationLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>Command</string>
        </property>
        <property name="buddy">
         <cstring>commandLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="commandLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="cEffortLabel">
        <property name="text">
         <string>C Effort</string>
        </property>
        <property name="buddy">
         <cstring>cEffortLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="cEffortLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dataOperations">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Operations</string>
    </property>
    <widget class="QPushButton" name="zeroPosButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Zero Position Sensor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="runRegressionButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Linear Regression</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionDevice_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>App Options</string>
   </property>
  </action>
  <action name="actionDevice_Settings">
   <property name="text">
    <string>Device Settings</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>frequencySlider</tabstop>
  <tabstop>amplitudeSpinBox</tabstop>
  <tabstop>saveLogButton</tabstop>
  <tabstop>controlModeBox</tabstop>
  <tabstop>timestampLineEdit</tabstop>
  <tabstop>positionLineEdit</tabstop>
  <tabstop>currentLineEdit</tabstop>
  <tabstop>accelerationLineEdit</tabstop>
  <tabstop>commandLineEdit</tabstop>
  <tabstop>cEffortLineEdit</tabstop>
  <tabstop>zeroPosButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>frequencySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>frequencyDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
